pages/proposals.jsx

```diff
@@ -2,7 +2,6 @@ var React = require('react');
 var Header = require('../components/header.jsx');
 var Footer = require('../components/footer.jsx');
 var Jumbotron = require('../components/jumbotron.jsx');
-var Icon = require('react-fa');
```

Comment:
This package no longer works for some reason. We don't really need this anyway so I replaced those `<Icon>` with `<span className="fa fa-spinner></span>`.

Note: this file is currently not in use. In the next few days I will replace everything here with a new proposal form (using `react-formbuilder`)